src/main/deploy/pathplanner/New Path.path

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
We should take all of the files from `src/main/deploy/pathplanner` and remove them from this branch since we don't want them in `main` if they're not tested. We should create a separate `pathplanner` branch where they can stay while we test them.